docs/how-to/craft/configure-distribution-information.rst

```diff
@@ -0,0 +1,351 @@
+.. _how-to-configure-distribution-information:
+
+Configure distribution information
```

Comment:
This is an attempt at a more meaningful term than "metadata". Happy to take alternative suggestions.

Reply:
"package information" or "package metadata" may work.  Distribution is good, but it may be conflated with Linux distributions.

Reply:
I like "package", thanks. I'll rename the page after all the other changes are in.

Reply:
5564297